pyxem/signals/diffraction2d.py

```diff
@@ -997,9 +998,12 @@ def get_direct_beam_position(self, method, lazy_result=False, **kwargs):
 
     def center_direct_beam(
         self,
-        method,
+        method=None,
         half_square_width=None,
+        shifts=None,
```

Comment:
I think we should find some way of telling users that passing `method` and `shifts` will be a bad idea, even if that's only via docstrings.

Reply:
Agreed. I added a check in the beginning of the function, which will raise an error if either both or none of the parameters (`method` and `shifts`) are specified.